RFCs/FS-1126-allow-lower-case-du-cases when-require-qualified-access-is specified.md

```diff
@@ -6,7 +6,7 @@ This RFC covers the detailed proposal for this suggestion.
 
 - [x] [Suggestion]((https://github.com/fsharp/fslang-suggestions/issues/131)
```

Comment:
Typo, there’s an extra opening parentheses here (was already there, but maybe you can fix it as part of this PR).

Reply:
Fixed